### PR TITLE
rkt: Don't warn on empty pod logs

### PR DESCRIPTION
**Pull request by euank**
If a pod has not printed anything to stdout/stderr, it's expected
behaviour to get `-- No entries --`, even when requesting json output.
Prior to this change, a warning would be printed in such an occasion.

cc @yifan-gu

### CODE REVIEW

File: pkg/kubelet/rkt/log.go [hunk 1]
```diff
@@ -52,9 +55,12 @@ func pipeLog(wg *sync.WaitGroup, logOptions *api.PodLogOptions, r io.ReadCloser,
 	for scanner.Scan() {
 		var data interface{}
 		b := scanner.Bytes()
+		if bytes.Equal(b, journalNoEntriesLine) {
+			continue
+		}
 
 		if err := json.Unmarshal(b, &data); err != nil {
-			glog.Warningf("rkt: Cannot unmarshal journal log, skipping line: %v", err)
+			glog.Warningf("rkt: Cannot unmarshal journal log '%q', skipping line: %v", string(b), err)
```
Comment (yifan-gu) [inline pkg/kubelet/rkt/log.go]: Do you really need the quotes when you already use `%q` ?

File: pkg/kubelet/rkt/log.go [hunk 2]
```diff
@@ -142,6 +148,7 @@ func (r *Runtime) GetContainerLogs(pod *api.Pod, containerID kubecontainer.Conta
 		}
 	}
 
+	glog.V(4).Infof("rkt: gettings logs with command '%v'", cmd.Args)
```
Comment (yifan-gu) [inline pkg/kubelet/rkt/log.go]: Maybe s/'%v'/%q ?

**Comment by euank** [comment]
@yifan-gu changed as suggested, thanks

**Comment by yifan-gu** [comment]
LGTM

**Pull request merged**